comfy_api_nodes/nodes_rodin.py

```diff
@@ -229,8 +228,8 @@ def get_quality_mode(self, poly_count):
 
         return mesh_mode, quality
 
-    async def download_files(self, url_list):
-        save_path = os.path.join(comfy_paths.get_output_directory(), "Rodin3D", datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S"))
+    async def download_files(self, url_list, task_uuid):
+        save_path = os.path.join(comfy_paths.get_output_directory(), f"Rodin3D_{task_uuid}")
```

Comment:
to change the behavior less, maybe better to leave the "Rodin3D" folder?

`os.path.join(comfy_paths.get_output_directory(), f"Rodin3D_{task_uuid}")`
->
`os.path.join(comfy_paths.get_output_directory(), "Rodin3D", str(task_uuid))`